packages/zipkin-instrumentation-restify/src/restifyMiddleware.js

```diff
@@ -33,8 +33,14 @@ module.exports = function restifyMiddleware({tracer, serviceName, port = 0}) {
         res.removeListener('close', onCloseOrFinish);
         res.removeListener('finish', onCloseOrFinish);
 
+        let error = null;
```

Comment:
It's better to do this immutably;

```
const error = res.statusCode >= 400 ? res.statusCode : null;
```

And maybe also better to move it inside the `tracer.scoped(() => ` block (after what looks like line 42), so it's defined closer to where it's used?

Reply:
Sure, I'll make it immutable and inside the scoped block. Do you want it to handle codes <200 like how Brave does, or keep it to being 400 and greater only?